server/issue.go

```diff
@@ -168,20 +168,24 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 		return http.StatusInternalServerError, errors.Errorf("Failed to create issue. %s", err.Error())
 	}
 
-	// Reply to the post with the issue link that was created
+	// Reply with an ephemeral post with the Jira issue formatted as slack attachment.
 	reply := &model.Post{
 		Message:   fmt.Sprintf("Created a Jira issue %v/browse/%v", ji.GetURL(), created.Key),
 		ChannelId: channelId,
 		RootId:    rootId,
 		ParentId:  rootId,
-		UserId:    mattermostUserId,
+		UserId:    ji.GetPlugin().getConfig().botUserID,
 	}
-	_, appErr = api.CreatePost(reply)
-	if appErr != nil {
+
+	attachment, err := ji.GetPlugin().getIssueAsSlackAttachment(ji, jiraUser, created.Key)
+	if err != nil {
 		return http.StatusInternalServerError,
-			errors.WithMessage(appErr, "failed to create notification post "+create.PostId)
+			errors.WithMessage(err, "failed to create notification post "+create.PostId)
 	}
 
+	reply.AddProp("attachments", attachment)
+	_ = api.SendEphemeralPost(mattermostUserId, reply)
```

Comment:
@aaronrothschild Do we want to show the other people in the channel this formatted output? Currently, this PR's code changes the output to be an ephemeral post instead of a public post.

@iamsayantan I'm thinking we should post the expanded version to the channel. Either that, or keep the original functionality in tact, and post the expanded one with an ephemeral post. Could you post a screenshot with the ticket's description collapsed (so it says "Show More"), to show a different context?

I'll defer to @aaronrothschild on the decision.

Reply:
@mickmister The Jira ticket mentioned to create an ephimeral post.

> Change the `/jira create` feature to also show the “view issue” of the newly created issue as an ephemeral message visible to the person who just created the issue (and also display the URL to Jira).

Reply:
Thanks for pointing this out. What do you think about creating a public post and an ephemeral post? The public post could be:

> Created an issue (issue key) - `(summary)`

Reply:
Okay, I will work on it and then decide if it looks good.

Reply:
@mickmister Have a look and let me know your thoughts. If this looks good, I'll update the pull request.

![issue_message](https://user-images.githubusercontent.com/19422764/74096508-043a6e00-4b26-11ea-8776-a59a1c6536b0.jpg)

And this one is with colapsed issue description.
![long_issue](https://user-images.githubusercontent.com/19422764/74096515-19af9800-4b26-11ea-89f7-f31810338f8e.jpg)

Reply:
I like this approach.  @iamsayantan, in the permanent post, can you link to the ticket, similar to the ephemeral post does?